src/api/server/common.rst

```diff
@@ -1586,6 +1586,76 @@ node, you can use:
 This returns an entire statistics object, as with the full request, but
 containing only the requested individual statistic.
 
+==================================
+``/_node/{node-name}/_prometheus``
+==================================
+
+.. http:get:: /_node/{node-name}/_prometheus
+    :synopsis: Returns server statistics in prometheus format
+
+    The ``_prometheus`` resource returns a text/plain response that consolidates our
+    `_stats`, `active_task`, and `_system` endpoints. The format is determined by
```

Comment:
```suggestion
    ``_stats``, ``active_task``, and ``_system`` endpoints. The format is determined by
```

Maybe link to the docs about these endpoints, too?

Reply:
+1 to Flimzy's comment here